### PR TITLE
Use annotation to pause chaos

### LINKED ISSUE #447: Use annotation of Chaos Object to pause experiment
## Feature Request

**Is your feature request related to a problem? Please describe:**

At present, If we want to pause an experiment, we need patch this object, such as: 

```
kubectl patch podchaos pod-kill-example --namespace chaos-testing --type merge --patch 'spec:
 paused: false'
``` 
I think this operation is complicated and incomprehensible. 

**Describe the feature you'd like:**

I recommend that we can use annotation of the object to pause experiment, in the way, we can pause the experiment using the following command: 

```
kubectl annotate podchaos pod-kill-example -n chaos-testing pause='true'
```

### PR DESCRIPTION
**Pull request by yeya24**
Signed-off-by: yeya24 <yb532204897@gmail.com>

### What problem does this PR solve? 

Fixes #447

### CODE REVIEW

File: api/v1alpha1/common_types.go [hunk 1]
```diff
@@ -21,6 +21,8 @@ import (
 	ctrl "sigs.k8s.io/controller-runtime"
 )
 
+const PauseAnnotationKey = "admission-webhook.pingcap.com/pause"
```
Comment (cwen0) [inline api/v1alpha1/common_types.go:24]: This feature isn't related to `admission-webhook`, I think we can use the other key, eg: "experiment.pingcap.com/pause` or others.

Comment (yeya24) [reply api/v1alpha1/common_types.go:24]: Make sense.

**Pull request merged**